lib/rust/ensogl/core/src/timer/timeout.rs

```diff
@@ -0,0 +1,84 @@
+//! FRP bindings for browser `setTimeout` timer.
+
+use crate::prelude::*;
+
+use enso_web::traits::ClosureOps;
+use enso_web::window;
+use enso_web::Closure;
+
+use std::time::Duration;
+
+/// Closure type alias for use in `setTimeout` call.
+type TimerClosure = Closure<dyn FnMut()>;
+
+/// A one-shot timer based on `setIntevral` browser API.
+///
+/// The timer can be started or stopped at any time. The timer value
+/// is provided during the start.
+///
+/// When the timer expires, `expired` is emitted.
+#[derive(Clone, CloneRef, Debug)]
+pub struct Timeout {
+    /// Starts the timer immediately with provided timeout value.
+    /// If the timer was already started, it is cancelled and restarted.
```

Comment:
Why these lines are split so strangely? We have limit of 100 chars. Please use it everywhere.